tools/tck-api/src/main/scala_2.13/org/opencypher/tools/tck/api/events/Collections.scala

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Why is this added by this PR? Also, tangentially and at a glance, the whole thing feels hacky.

Reply:
The respective methods disappeared in Scala 2.13 without real replacement.